README.md

```diff
@@ -1,2 +1,156 @@
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+  <img src="./src/asset/img/miniNetflix.png" width="auto">
+
+# Mini-Netflix-Website <a name="about-project"></a>
+
+We built a Mini-Netflix-Website that displays several Tv shows from an external APITV Shows API App.This is a JavaScript-based application that allows users to explore and interact with various TV shows. Users can view details about different TV shows, like the comments and the likes.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+-**HTML & CSS**
+
+-**JAVASCRIPT**
+
+-**WEBPACK**
+
+-**JEST**
+
+### Key Features <a name="key-features"></a>
+
+- **TV Shows Listing: Browse a collection of popular TV shows with their details, such as title, description, and airing date.**
+
+- **Like Shows: Users can like their favorite TV shows, and the app will keep track of their preferences.**
+
+- **Comments: Users can leave comments on TV shows to share their opinions and discuss with other users.**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Visit our live project]()
+
+## 🚀 Description Video<a name="video-demo"></a>
+
+- [Link to our description video](https://drive.google.com/file/d/17jh4JKd81T7QvOJDLy_CoZUjgQS_UlN_/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+sudo apt install git-all
+or
+npm install
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone  https://github.com/Ayokunnumi1/Mini-Netflix-Websit.git
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ayoknnumi**
+
+- GitHub: [@Ayokunnumi1][https://github.com/Ayokunnumi1]
+- Twitter: [@Ayokunnumi1] [https://twitter.com/AyokunnumiA]
+- LinkedIn: [Ayokunnumi] [https://www.linkedin.com/in/ayokunnumiomololu]
+
+👤 **Hydaline**
+
+- GitHub: [https://github.com/hydaline-code]
+- Twitter: [@hydalineC][https://twitter.com/@hydalineC]
+- LinkedIn: [https://www.linkedin.com/in/charlene-djoungang-0851aa21a/]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **[Add a About page ]**
+
+- **[Add more interactivity ]**
+
+- **[Add a contact form page ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/https://github.com/Ayokunnumi1).
```

Comment:
- [ ] [OPTIONAL] Kindly change the issues link as this attached link is currently broken. The issue link would be used by any developer that discovers and wants to raise an issue concerning the project.

Reply:
Ok thank you, I have worked on it.